docs/src/pages/Conformance.md

```diff
@@ -272,8 +272,14 @@ The IBM FHIR Server does not consider the `Quantity.comparator` field as part of
 ### Searching on URI
 URI searches on the IBM FHIR Server are case-sensitive with "exact-match" semantics. The `above` and `below` prefixes can be used to perform path-based matching that is based on the `/` delimiter.
 
+There is one exception to the statement above. The `url` search parameter, which is defined in the base FHIR specification on definitional resource types as a URI search parameter, is actually treated as a canonical search parameter of type REFERENCE, as documented in the [FHIR specification](http://build.fhir.org/references.html#canonical). The following section of this document describes how the IBM FHIR Server processes canonical reference searches.
+
 ### Searching on Reference
-Reference searches on the IBM FHIR Server support search on:
+Reference searches on the IBM FHIR Server support search on elements of type Reference and on elements of type canonical.
+
+**Search on Elements of Type Reference:**
+
+Reference searches may be of the following types:
 
 * a relative reference - `1` where it is reflexsively determined to be a subset of possible targets, such as `Patient/1`, `Group/1`
```

Comment:
unrelated to the PR but I wonder whether that is supposed to say `reflexively` or `reflectively` :-)

Reply:
I think reflectively makes more sense in this context, so changed it to that.